inputControl-report/bind-with-report/demo.html

```diff
@@ -3,6 +3,5 @@
 <div id="container">
     <div id="ic">
         <div><p>Loading...</p></div>
-    </div>
-    <div id="report"></div>
-</div>
\ No newline at end of file
+</div><div id="report"></div>
```

Comment:
looks like formatting has been lost here. maybe it's ok, just want you to know

Reply:
I've updated this.